src/styles/variables.css

```diff
@@ -0,0 +1,11 @@
+/* #region variables */
+
+:root {
+  --white: #fff;
+  --border: #f3f3f3;
+  --text1: #616070;
+  --text2: #060b35;
```

Comment:
Give more meaningful names to variables. text1 and text2 are not descriptive enough